packages/style-engine/class-wp-style-engine.php

```diff
@@ -278,29 +278,6 @@ protected static function is_valid_style_value( $style_value ) {
 		return true;
 	}
 
-	/**
-	 * Private constructor to prevent instantiation.
-	 */
-	private function __construct() {
-		// Register the hook callback to render stored styles to the page.
-		static::register_actions( array( __CLASS__, 'process_and_enqueue_stored_styles' ) );
-	}
-
-	/**
-	 * Utility method to retrieve the main instance of the class.
-	 *
-	 * The instance will be created if it does not exist yet.
-	 *
-	 * @return WP_Style_Engine The main instance.
-	 */
-	public static function get_instance() {
-		if ( null === static::$instance ) {
-			static::$instance = new static();
-		}
-
-		return static::$instance;
```

Comment:
Nice that we no longer need the instance! Just noticed that we can remove the declaration at line 31 now, too? `private static $instance = null;`